Tests/UnitTests/GridSystemData2Tests.cpp

```
@@ -1,13 +1,60 @@
 #include <Core/Grid/VertexCenteredVectorGrid.hpp>
 
+#include <utility>
+
 using namespace CubbyFlow;
 
+namespace
+{
+std::vector<uint8_t> MakeGridSystemDataWithSecondVelocity()
+{
+    FaceCenteredGrid2 velocity0({ 2, 3 }, {}, {}, { 1.0, 2.0 });
+    FaceCenteredGrid2 velocity1({ 2, 3 }, {}, {}, { 3.0, 4.0 });
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If MANUALLY_ALIGNED_STRUCT is a macro then please configure it.

(unknownMacro)